crates/walrus-orchestrator/src/main.rs

```diff
@@ -38,8 +38,13 @@ type ClientParameters = ProtocolClientParameters;
 
 /// The orchestrator command line options.
 #[derive(Parser, Debug)]
-#[command(author, version, about = "Testbed orchestrator", long_about = None)]
-#[clap(rename_all = "kebab-case")]
+#[command(
+    author, 
+    version, 
+    about = "Testbed orchestrator", 
```

Comment:
Nit (trailing whitespace):
```suggestion
    author,
    version,
    about = "Testbed orchestrator",
```